pkg/nbcparser/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/Azure/agentbaker/pkg/nbcparser
```

Comment:
we can probably move this to the top-level and rename the module

e.g. `module github.com/Azure/agentbaker/nbcparser` since it's own module, and not an AgentBaker pkg